server/README.md

```diff
@@ -463,6 +463,13 @@ The special value `all` enables all statistics.
 
 Once enabled, you should see the metrics values changing on the `/metrics` endpoint for the management endpoint.
 
+### Setting additional keycloak profile options
+
+For setting additional keycloak profile options at startup of the docker container, you can set the JAVA_TOOL_OPTIONS environment variable.
```

Comment:
I believe there's better/easier way to do it - pass it as container arguments. Here's an example: 

```
docker run quay.io/keycloak/keycloak:latest -Dkeycloak.profile.feature.scripts=enabled -Dkeycloak.profile.feature.upload_scripts=enabled
```

Using `JAVA_TOOL_OPTIONS` might work but it's a very "hacky" approach.

Reply:
Are you sure that the arguments are passed to the keycloak entrypoint command? If I remember, this didn't work

Reply:
Yes, I'm pretty sure - I've just checked it. Since UBI Minimal doesn't contain `ps` command, I had to navigate to `/proc/193` and look at `cat cmdline`. Here's what I got:

```
Terminal1: docker run -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin quay.io/keycloak/keycloak:latest -Dkeycloak.profile.feature.scripts=enabled -Dkeycloak.profile.feature.upload_scripts=enabled

Terminal2: cat cmdline 
java-D[Standalone]-server-Xms64m-Xmx512m-XX:MetaspaceSize=96M-XX:MaxMetaspaceSize=256m-Djava.net.preferIPv4Stack=true-Djboss.modules.system.pkgs=org.jboss.byteman-Djava.awt.headless=true--add-exports=java.base/sun.nio.ch=ALL-UNNAMED--add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED--add-exports=jdk.unsupported/sun.reflect=ALL-UNNAMED-Dorg.jboss.boot.log.file=/opt/jboss/keycloak/standalone/log/server.log-Dlogging.configuration=file:/opt/jboss/keycloak/standalone/configuration/logging.properties-jar/opt/jboss/keycloak/jboss-modules.jar-mp/opt/jboss/keycloak/modulesorg.jboss.as.standalone-Djboss.home.dir=/opt/jboss/keycloak-Djboss.server.base.dir=/opt/jboss/keycloak/standalone-Djboss.bind.address=172.17.0.2-Djboss.bind.address.private=172.17.0.2-c=standalone-ha.xml-Dkeycloak.profile.feature.scripts=enabled-Dkeycloak.profile.feature.upload_scripts=enabledbash-4.4$ ^C
```